go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0
-	github.com/ava-labs/avalanchego v1.10.18-rc.0
+	github.com/ava-labs/avalanchego v1.10.18-0.20231211033727-3c26d85f63c9
```

Comment:
can we use the same tag for `1.10.123-refactor-client-1`?

Reply:
I'm using a commit hash now since I think the tags don't work with e2e.